pkg/universe.dagger.io/go/test.cue

```diff
@@ -2,13 +2,30 @@ package go
 
 // Test a go package
 #Test: {
-	// Package to test
-	package: *"." | string
+	// DEPRECATED: use packages instead
+	package: string | *null
+
+	// Packages to test
+	packages: [...string]
 
 	#Container & {
 		command: {
+			_packages: [...string] | []
```

Comment:
Nit: `| []` should be useless

Reply:
clearly. Some debugging code I didn't remove…